packages/expo-video/src/VideoView.tsx

```diff
@@ -84,3 +90,43 @@ function getPlayerId(player: number | VideoPlayer): number | null {
   }
   return null;
 }
+
+/**
+ * Returns a shared object, which is automatically cleaned up when the component is unmounted.
+ */
+// TODO: when SharedObject type is added make T extend it
+export function useReleasingSharedObject<T>(factory: () => T, dependencies: DependencyList): T {
```

Comment:
Let's not export it yet. We'll probably move this to `expo-modules-core` and export it from there instead.

```suggestion
function useReleasingSharedObject<T>(factory: () => T, dependencies: DependencyList): T {
```